components/header-nav.vue

```diff
@@ -67,17 +67,17 @@ export default {
     BurgerMenuNav,
     OrganizationNav
   },
+  data() {
+    const showFrenchGovLogo = this.$i18n.locale === 'fr-fr'
+    return { showFrenchGovLogo }
```

Comment:
Il me semble que la Marianne n'est pas le logo du gouvernement français 

```suggestion
    const showTheMarianne = this.$i18n.locale === 'fr-fr'
    return { showTheMarianne }
```

Reply:
c'est bien dans la charte graphique du gouvernement : https://www.gouvernement.fr/charte/charte-graphique-les-fondamentaux/le-bloc-marque

Reply:
Effectivement, sous cette forme c'est le bloc marque du gouvernement